cmd/opampsupervisor/e2e_test.go

```diff
@@ -31,12 +30,13 @@ import (
 	"github.com/open-telemetry/opamp-go/protobufs"
 	"github.com/open-telemetry/opamp-go/server"
 	"github.com/open-telemetry/opamp-go/server/types"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/cmd/opampsupervisor/supervisor"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/cmd/opampsupervisor/supervisor/config"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	semconv "go.opentelemetry.io/collector/semconv/v1.21.0"
 	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/cmd/opampsupervisor/supervisor"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/cmd/opampsupervisor/supervisor/config"
```

Comment:
Not sure the linter wasn't complaining about it

Reply:
I believe the linter only runs against files which match certain tags. See https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/11557

Not sure if the new syntax is handled more appropriately but it's possible we have better coverage now.

Reply:
Probably not. This change was triggered by my IDE, not linter